client/Components/App.jsx

```diff
@@ -1,10 +1,17 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import OAuth from './OAuth'
 
 class App extends Component {
-    state = {  }
     render() {
-        return <h1>Success! The app loaded.</h1>
+        return (
+            <>
+                <OAuth />
+            </>
+        )
     }
 }
 
-export default App
+const mapStateToProps = () => {}
+
+export default connect(mapStateToProps)(App)
```

Comment:
is preparation for future use case the reason why we added mapState even though it doesn't return anything? we can just pass `null` too, for now.